claat/proto-renderer/html/render.go

```diff
@@ -0,0 +1,37 @@
+package html
+
+import (
+	"go/build"
+	"io"
+	"path/filepath"
+	"strings"
+	"text/template"
+
+	"github.com/googlecodelabs/tools/claat/proto-renderer"
+)
+
+const (
+	tmplsRltvDir = "src/github.com/googlecodelabs/tools/claat/proto-renderer/html/templates/*"
+)
+
+var (
+	tmplsAbsDir = filepath.Join(build.Default.GOPATH, tmplsRltvDir)
+	t           = template.Must(template.New("html").ParseGlob(tmplsAbsDir))
+)
+
+// Render returns the rendered HTML representation of a devrel_tutorial proto,
```

Comment:
Remove "devrel_"

Reply:
If so, it'll it implies it only renders `devrel_tutorial.Tutorial`. Trying to make explicit that it renders _any_ proto from the `devrel_tutorial` package